src/mocks/timeseriesDataPointsList.ts

```diff
@@ -34,6 +113,56 @@ export function randomData(
   return { datapoints: data, id: 1337 };
 }
 
+export function randomData(
+  start: number,
+  end: number,
+  n: number,
+  config: MockChartDataConfig,
+  granularity?: number
+): DatapointsGetAggregateDatapoint {
+  const data = [];
+  const { min, max, continousDeviation } = config;
+  const dt = granularity ? granularity : (end - start) / n;
+  let prevMax = (min + max) / 2;
+  for (let i = start; i <= end; i += dt) {
+    const values = getRandomDeviation(min, max, continousDeviation, prevMax);
+    prevMax = values[2];
+    data.push({
+      timestamp: new Date(i),
+      average: values[1],
+      min: values[0],
+      max: values[2],
+      count: 7000,
+    });
+  }
+
+  return { datapoints: data, id: 1337 };
+}
+
+function getRandomDeviation(
+  min: number,
+  max: number,
+  continousDevition: number,
```

Comment:
a typo – `continousDevition` -> `continuousDeviation` and other words with `continuous`